src/main/java/org/openstreetmap/atlas/geography/Polygon.java

```diff
@@ -714,4 +718,29 @@ private void verifyVertexIndex(final int index)
             throw new CoreException("Invalid Vertex Index {}.", index);
         }
     }
+
+    /**
+     * This method will return cartesian equivalent coordinates for earth coordinates
+     * 
+     * @return Tuple which represents X and Y on catesian plane
+     * @see <a href=
+     *      "https://www.grasshopper3d.com/forum/topics/best-way-to-translate-latitude-longitude-data-into-xyz-points?commentId=2985220%3AComment%3A1804229"></a>
+     * @see <a href=
+     *      "https://en.wikipedia.org/wiki/Geographic_coordinate_conversion#From_geodetic_to_ECEF_coordinates"></a>
+     * @see <a href="https://en.wikipedia.org/wiki/Earth_radius"></a>
+     */
+    private Tuple<Double, Double> getCartesianCoordinates(final Latitude latitude,
```

Comment:
Instead of Doubles, consider using `EarthCenteredEarthFixedCoordinate`.

Reply:
You could add this as a static method to that class also.

Reply:
Actually, I don't even need a separate method here, directly used ```EarthCenterEarthFixedCoordinate``` object in the ```isClockwise``` method